src/pages/[platform]/build-a-backend/auth/connect-your-frontend/sign-in/index.mdx

```diff
@@ -1103,13 +1107,148 @@ func socialSignInWithWebUI() -> AnyCancellable {
 
 Your application's users can also sign in using passwordless methods. To learn more, visit the [concepts page for passwordless](/[platform]/build-a-backend/auth/concepts/passwordless/)
 
+<InlineFilter filters={["angular", "javascript", "nextjs", "react", "react-native", "vue"]}>
+
+### New auth flow type
+
+In order to facilitate the new passwordless sign in options, Cognito is introducing a new auth flow type known as `USER_AUTH`. This flow is designed to be flexible and supports both password and passwordless sign in factors.
+
+```ts
+const { nextStep } = await signIn({
+username: state.username,
+   options: {
+        authFlowType: "USER_AUTH",
+   }
+});
+```
+
+### Sign In with a Preferred Challenge for First Factor
+
+The USER_AUTH sign in flow will support the following methods of first factor authentication: `WEB_AUTHN`, `EMAIL_OTP`, `SMS_OTP`, `PASSWORD`, and `PASSWORD_SRP`.
+
+If the desired first factor is known before the sign in flow is initiated it can be passed to the initial sign in call:
+
+```ts
+type AuthFactorType = 
+    | 'WEB_AUTHN' 
+    | 'EMAIL_OTP' 
+    | 'SMS_OTP' 
+    | 'PASSWORD' 
+    | 'PASSWORD_SRP';
+
+type SignInOptions = AuthServiceOptions & {
+    authFlowType?: AuthFlowType;
+    clientMetadata?: ClientMetadata;
+    preferredChallenge?: AuthFactorType;
+};
```

Comment:
It might be confusing to have types here that aren't referenced in the examples; the connection between these types and how to pass as options is unclear. Is there a way we can clarify this? Maybe using the `SignInInput` interface which is publicly exposed by the library... or just relying on the examples?